### PR TITLE
[Tizen/Api] support tensorflow model in singleshot

### PR DESCRIPTION
**Pull request by jaeyun-jung**
1. add tensorflow pipeline in singleshot
2. testcase with tensorflow speech cmd model

Signed-off-by: Jaeyun Jung <jy1210.jung@samsung.com>

### CODE REVIEW

File: tizen-api/src/tizen-api-util.c [hunk 1]
```diff
@@ -0,0 +1,442 @@
...
+}
+
+/**
+ * @brief Checks the availability of the given execution environments.
+ */
+int
+ml_util_check_nnfw (ml_nnfw_e nnfw, ml_nnfw_hw_e hw)
+{
+  /** @todo fill this function */
+#ifndef ENABLE_TENSORFLOW_LITE
```
Comment (myungjoo) [inline tizen-api/src/tizen-api-util.c]: You cannot determine this at build time.
If someone installs tensorflow-lite subplugin (built separatedly with /ext/nnstreamer/...), you get this supported.
If someone deletes the subplugin, you cannot support it.

You may want to use ```get_subplugin``` API (nnstreamer_subplugin.h).

Comment (jaeyun-jung) [reply tizen-api/src/tizen-api-util.c]: Updated to check tensorflow using nnstreamer_filter_find()

File: api/capi/src/nnstreamer-capi-single.c [hunk 2]
```diff
@@ -149,6 +149,54 @@ ml_single_open (ml_single_h * single, const char *model_path,
...
+        str_type = gst_tensors_info_get_types_string (&out_info);
+        str_name = gst_tensors_info_get_names_string (&out_info);
+        out_option = g_strdup_printf ("output=%s outputtype=%s outputname=%s",
+            str_dim, str_type, str_name);
+        g_free (str_dim);
+        g_free (str_type);
+        g_free (str_name);
+
+        pipeline_desc =
+            g_strdup_printf
```
Comment (kparichay) [inline api/capi/src/nnstreamer-capi-single.c]: LGTM.

Comment (myungjoo) [reply api/capi/src/nnstreamer-capi-single.c]: you need to click "approve". GitHub doesn't recognize your "LGTM".

**Pull request merged**